### PR TITLE
Update changelog for Fields page performance

### PR DESCRIPTION
**Pull request by bpuckhaber**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 2022-11-17
+| What's new | 
+| :--------- |
+| We have improved the performance of the Fields page to better support graphs with a large number of fields (> 1000).
```
Comment (timbotnik) [inline CHANGELOG.md:4]: I'd say if we include this change that we should be super explicit about fixing _part_ of the problem but not all of the problem, and that we're working on solutions to better handle the other issues.  Alternatively, wait until we've finished the improvements to handling the stats loading / errors UX and announce them together.

**Review by nkhi — CHANGES_REQUESTED** [review]
I think it should be

"We have improved the scrolling performance of the Fields page for graphs with over one thousand fields. Additionally, timeout requests on the field page for graphs with over five thousand fields omit operation usage and field execution metrics from the page."

My two cents

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 2022-12-05 Fields Page Performance Enhancements
+| What's new | 
+| :--------- |
+| We have improved the performance of the Fields page to better support graphs with a large number of fields (> 1000). Also, Fields page request timeouts experienced by large graphs (> 5000 fields) are now handled more gracefully and instead omit operation usage and field execution metrics from the page.
```
Comment (timbotnik) [inline CHANGELOG.md:4]:
```suggestion
| We have improved the performance of the Fields page to better support graphs with a large number of fields. 
 For the largest of graphs (> 5000 fields) we now omit usage metrics and encourage you to navigate to [individual field pages](https://www.apollographql.com/blog/backend/performance/field-level-insights-in-studio/) for field-level insights.
```

**Review by timbotnik — APPROVED** [review]
Updated with some changes!

**Comment by nkhi** [comment]
I think omitting the word `scrolling` from `performance` in the first sentence makes it unclear to me. Everything else LGTM. Your call!

**Review by nkhi — APPROVED** [review]

**Comment by nkhi** [comment]
I wonder if we could use Astros pagination to bring back in-list-metrics for >5000 field schemas?

**Pull request merged**